src/passkit.cs

```diff
@@ -1571,11 +1626,65 @@ public enum PKDisbursementRequestSchedule : long {
 	[BaseType (typeof (NSObject))]
 	interface PKDisbursementRequest {
 
+#if XAMCORE_5_0
+		[Export ("currencyCode")]
+#else
 		[NullAllowed, Export ("currencyCode")]
+#endif
 		string CurrencyCode { get; set; }
 
+#if XAMCORE_5_0
+		[Export ("summaryItems", ArgumentSemantic.Copy)]
+#else
 		[NullAllowed, Export ("summaryItems", ArgumentSemantic.Copy)]
+#endif
```

Comment:
I have mixed feelings about this two #if.. in one hand I do understand them, we moved from 'string?' to 'string'. Peres is not an API breaking change, and code will compile unless the developer has the nullability warnings sets as  errors. 

So the removal of the ? will move use from a possible  runtime error (we passed null) to a compiler warning possible error.  I prefer users to get a warning than a runtime error in their applications when they are deployed to iOS 15 and later, specially when we do not have a clear date for XAMCORE_5_0.

@dalexsoto @rolfbjarne  thoughts?

Reply:
The other side is that if the API is actually nullable (for instance by accident on Apple's side), removing the nullability on our side will now result in a C# exception (ArgumentNullException) when the code previously worked.

Reply:
CurrencyCode might have not been an accident based on the documentation from the service https://developer.apple.com/documentation/passkit/apple_pay/payment_token_format_reference?language=objc

```
currencyCode | string | ISO 4217 numeric currency code, as a string to preserve leading zeros
-- | -- | --
```

And it uses https://www.iso.org/iso-4217-currency-codes.html which AFAIK does not have a way to represent null. 

Similar to the summary items https://developer.apple.com/documentation/passkit/pkdisbursementrequest/3112810-summaryitems?changes=l_4&language=objc
```
This array must contain one [PKPaymentSummaryItem](https://developer.apple.com/documentation/passkit/pkpaymentsummaryitem?changes=l_4&language=objc), which contains the amount the individual receives.
```

If we allow null values, the requests to the payment service won't be valid at all.

Reply:
Either way works for me :)

Reply:
Sounds good! I went ahead and removed those Xamcore conditionals! @rolfbjarne @mandel-macaque